src/Sarif.Viewer.VisualStudio/Sarif/CodeFlow.extensions.cs

```diff
@@ -38,5 +34,17 @@ public static AnnotatedCodeLocationCollection ToAnnotatedCodeLocationCollection(
 
             return model;
         }
+
+        public static CallTree ToCallTree(this CodeFlow codeFlow)
+        {
+            if (codeFlow == null || codeFlow.Locations.Count == 0)
```

Comment:
>codeFlow == null || codeFlow.Locations [](start = 16, length = 38)

hm, does the following not work for this? also, what does our schema say about ensuring Locations is non-null? Does it make sense for it to be non-null but empty?

if (codeFlow?.Locations.Count == 0)